api/src/pcapi/core/fraud/models.py

```diff
@@ -310,12 +311,23 @@ class PhoneValidationFraudData(pydantic.BaseModel):
     phone_number: typing.Optional[str]
 
 
+class ProfileCompletionContent(pydantic.BaseModel):
+    activity: str
+    city: str
+    first_name: str
+    last_name: str
+    origin: str  # Where the profile was completed by the user. Can be the APP or DMS
+    postalCode: typing.Optional[str]
```

Comment:
pourquoi Optional ?

Reply:
Hmm on avait mis optional pour le code postal dans la strat tech
Côté DMSContent, c'est typé comme `Optional[str]`

Reply:
oui mais tu pourrais ajouter postalCode  dans le check `if all(elem is not None for elem in [activity, city, first_name, last_name]):` de `_create_profile_completion_fraud_check_from_dms` pour garantir que c'est non null

Reply:
Yes
Je vérifie que c'est bien récpéré dans les 2 formulaires DMS